src/MentionWrapper.js

```diff
@@ -43,20 +43,28 @@ class MentionWrapper extends Component {
     const tokens = textBeforeCaret.split(/\s/);
     const lastToken = tokens[tokens.length - 1];
 
-    // check if the text befor the caret ends with the last word
-    const triggerIdx = textBeforeCaret.endsWith(lastToken)
-      ? textBeforeCaret.length - lastToken.length
-      : -1;
+    // Get active trigger
+    let activeTrigger = 0;
+    this.triggers.forEach((trigger) => {
+      if (lastToken.includes(trigger)) {
```

Comment:
reason you need `includes` instead of `endsWith`?

Reply:
Because `lastToken` is the trigger AND the query string when you type. It will never end with the trigger. As soon as you get past the trigger and start typing the label name or assignee the dropdown will go away.